core/unit_test/CMakeLists.txt

```diff
@@ -415,10 +415,12 @@ KOKKOS_ADD_ADVANCED_TEST( UnitTest_PushFinalizeHook_terminate
   if(KOKKOS_ENABLE_LIBDL)
 
     KOKKOS_ADD_LIBRARY(
-    printer-tool SHARED
+    kokkosprinter-tool SHARED
     SOURCES tools/printing-tool.cpp
     )
```

Comment:
I think the issue is that we use `KOKKOS_ADD_LIBRARY`

Reply:
Is there an incantation of KOKKOS_ADD_LIBRARY that doesn't install?

Reply:
Or could we just remove https://github.com/kokkos/kokkos/commit/5621c99e1d011a2d5ec30ac1b8842e3eb4daa17a which changed this from add_library?